### ISSUE TITLE
Does anyone really dislike the __call__ syntax?

### ISSUE THREAD

**Issue by goodboy** [SIPp/pysipp#61]
I remember way back when first writing this we had an internal debate about `.run()` versus `__call__()` and can't remember why we settled on the latter.

In hindsight, after more years of codezing, I think the `.run()` is more clear and pythonic maybe?

Does anyone have opinions on this?

**Comment by kontsaki** [comment]
i agree that `.run()` is more explicit

do you lean on deprecating the scenario object being a callable?

**Comment by goodboy** [comment]
@kontsaki yeah, it use of `__call__()` was from a different "way" of design in younger years.

I wonder if `Scenario.run()` is just more simple and intuitive.

**Comment by kontsaki** [comment]
yes it feels like it, since there is a `.finalize()` coming as well.

**Comment by kontsaki** [comment]
hey @goodboy , what is the current way of running a single specific xml scenario? 
i used to work with uac/uas and set the `scen_file` attribute but apparently that was incorrect cause now it blows up with 
`Internal error, main_scenario already set` which i guess makes sense.

**Comment by kontsaki** [comment]
does it make sense to expose `ua` like `client` and `server` ?
